content/en/apps/reference/app-settings/token_login.md

```diff
@@ -0,0 +1,27 @@
+---
+title: ".token_login"
+linkTitle: ".token_login"
+weight: 5
+description: >
+  **Token login**: Instructions and schema for Login by SMS
+relatedContent: >
+  apps/reference/api#login-by-sms
+---
+
+Login via SMS settings are defined under the `token_login` key, as an object supporting the following properties:
+## `app_settings.json .token_login`
+| property         | type | required       | description                                                                                                                                                                              |
+|------------------|------|---------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| enabled | Boolean | yes | Enables or disables token_login deployment-wide. When this is false, users can't be updated to use token_login and any requests to login with a token link will fail.  |
+| translation_key | String | yes if `message` is not defined | Translation key for the information (helper) sms message that the user receives, along with their token-login link |
```

Comment:
Now that we don't have documentation for `message` we should just say the translation key is required.

```suggestion
| translation_key | String | yes | Translation key for the information (helper) sms message that the user receives, along with their token-login link |
```

Reply:
Oh oops!